docs/Catalog/Admin.md

```diff
@@ -64,10 +69,78 @@ The resulting permission set is equivalent to a union of all permissions
 provided by the policies attached to that role.
 
 
+### Extending built-in roles
+
+The initial "Source=Custom" roles defined by Quilt (`ReadQuiltBucket`, `ReadWriteQuiltBucket`) are locked to AWS ARNs, and thus cannot be extended by Quilt policies.  If you want a "Source=Quilt" UI-managed role that automatically has access to all registered buckets, you need to first import those specific polices.
+
+
+#### A. Create a Source=Quilt role
+
+For concreteness, we will create a new `UserReadQuiltBucket` read-only role to which we can add Quilt policies.
+
+1. Login to your Quilt instance at, e.g. https://quilt.mycompany.com
+2. Click on "Admin Settings" in the upper right, under your Profile name
+3. Scroll down to the "Roles" section on the bottom
+4. Click "+" to define a new role
+5. Name it `UserReadQuiltBucket`
+6. Save
+
+You can also create a `UserReadWriteQuiltBucket` if you want grant full permissions
+
+#### B. Display existing policies in AWS Console
+
+While still in the Roles section of Admin Settings:
+
+1. Find 'Custom' role "ReadWriteQuiltBucket"
+2. Click "Open in AWS Console" (square-with-arrow icon) on the right
+
+![](../imgs/admin-policy-aws-open.png)
+
+A new browser tab will open.
+You will see (at least) three polices, whose names include:
+* BucketReadPolicy
+* BucketWritePolicy
+* ReadQuiltPolicy
+
+![](../imgs/admin-policy-aws-list.png)
+
+3. Command-click those names to open each policy in its own tab, so you can copy the Policy ARN
+
+![](../imgs/admin-policy-arn.png)
+
+
+#### C. Import and attach policy ARNs
+
+In the initial window, while still in Admin Settings:
+
+1. Scroll down to Policies
+2. Click "+" to create a new policy
+3. Set Title to one of: `BucketReadPolicy` | `BucketWritePolicy` | `ReadQuiltPolicy`
+4. Click "set existing policy via ARN"
+5. For 'ARN' paste in "Policy ARN" from the corresponding browser tab (above)
```

Comment:
Uh, we should not be using ARNs that point to source=Custom or anything underneath it. Once again.

Here's what we should be recommending:
1. Create a Quilt policy with read/write/whatever access as desired (this is all drag and drop, with the GUI, and does not require separate instructions for read/write, it's literally an icon).